.storybook/manager.js

```diff
@@ -0,0 +1,69 @@
+// .storybook/manager.js
+import { addons } from "@storybook/addons";
+import { create } from "@storybook/theming/create";
+
+const lightTheme = create({
+  base: "light",
+  // Typography
+  fontBase: '"Open Sans", sans-serif',
+
+  //logo and Title
+  brandTitle: "Bitwarden Component Library",
+  brandUrl: "https://components.bitwarden.com",
```

Comment:
If testing on localhost or a Chromatic PR deployment, this will still link to `components.bitwarden.com`. Instead, we probably want to link to whatever the accurate home context is.

```suggestion
  brandUrl: "/",
```